yarn.lock

```diff
@@ -3210,16 +3210,16 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@types/babel__core@npm:^7.0.0, @types/babel__core@npm:^7.0.4, @types/babel__core@npm:^7.1.0, @types/babel__core@npm:^7.1.7":
-  version: 7.1.7
-  resolution: "@types/babel__core@npm:7.1.7"
+"@types/babel__core@npm:^7.0.0, @types/babel__core@npm:^7.0.4, @types/babel__core@npm:^7.1.0, @types/babel__core@npm:^7.1.7, @types/babel__core@npm:^7.1.9":
+  version: 7.1.9
```

Comment:
fwiw, it fails with 7.1.8 as well, so guessing it's https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44515

Reply:
Applying https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44515#discussion_r467392625 locally and removing the explicit type annotations leaves one error

![image](https://user-images.githubusercontent.com/1404810/89777178-7e66a200-db0b-11ea-9661-a9c9f779ba88.png)